### PR TITLE
Add cart & wishlist page

### PR DESCRIPTION
**Pull request by VLeads**
Add Cart & Wishlist page

Fix CSS alignment

Minor fixes

### CODE REVIEW

**Review by Developers-codz — APPROVED** [review]
Looking good to me . Can merge.

File: components/Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  
+}
+
+.cart-product-main{
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(47rem, max-content));
+    grid-gap: 1.5rem;
+    padding: 0;
+    justify-content: center;
+    /* display: flex;
```
Comment (SukanyaSen530) [inline components/Cart/cart.css:31]: Can remove commented lines.

**Review by SukanyaSen530 — APPROVED** [review]

File: components/Cart/cart.css (hunk 1 above)
Comment (VLeads) [reply components/Cart/cart.css:31]: Got it, will remove 👍

Comment (VLeads) [reply components/Cart/cart.css:31]: Got it, will remove 👍

Comment (VLeads) [reply components/Cart/cart.css:31]: Got it, will remove 👍

**Comment by VLeads** [comment]
Thanks @Developers-codz @SukanyaSen530 for reviews. Now merging.

**Pull request merged**